pkg/omf/functions/index/omf.index.repositories.fish

```diff
@@ -0,0 +1,60 @@
+function omf.index.repositories -d 'Manage package repositories'
+  # List repositories by default.
+  if not set -q argv[1]
+    set argv[1] list
+  end
+
+  switch $argv[1]
+    case help --help -h
+      omf help repositories
+      return
+
+    case list
+      for file in {$OMF_PATH,$OMF_CONFIG}/repositories
+        if test -f $file
+          read -z -l contents < $file
+          printf "$contents"
```

Comment:
I was thinking it was going to be more efficient. I just tried a `read` vs `cat` benchmark with 10,000 iterations and saw no difference on my machine, so maybe it isn't...